index.js

```diff
@@ -3832,20 +3832,27 @@ export const createContainerSpecLikeBom = async (path, options) => {
               console.log(`Parsing image ${img.image}`);
             }
             const imageObj = parseImageName(img.image);
+
             const pkg = {
-              name: imageObj.repo,
+              name: imageObj.name,
+              group: imageObj.group,
               version:
                 imageObj.tag ||
                 (imageObj.digest ? "sha256:" + imageObj.digest : "latest"),
               qualifiers: {},
-              properties: commonProperties
+              properties: commonProperties,
+              type: "container"
```

Comment:
Interesting. `container` is a process in execution, whereas this is an image. https://github.com/opencontainers/image-spec . Let me ask the community on slack.

Reply:
Steve clarified that `container` is indeed the correct type. We should read it in a `software` context.